### PR TITLE
Created secure-key-management.md

### LINKED ISSUE #72: Document secure use of Rekor pub key(s)
It would be great to have a quick and easy guide on this topic, including:

- Use of the API vs. TUF
- Locations of keys and certs
- Keys per shard/log in Rekor
- Commands to use to very the pubkey(s) (openssl, etc.)
- Verifying pubkey back up to the TUF community trust root (linking to article about how that root works wrt Sigstore/Rekor)

From @haydentherapper : "We should document that for the public deployment, the API should not be used to verify entries, that the keys should come from TUF. For private deployments, TUF is recommended but if you do choose to use the API, you should at least do trust-on-first-use." [Slack convo ref](https://sigstore.slack.com/archives/C01CX4E2K70/p1676498261172739)

### PR DESCRIPTION
**Pull request by ro4i7**
#### Summary
Close #72

### CODE REVIEW

**Comment by ro4i7** [comment]
@lkatalin please check this, if it can be merged.

File: content/en/rekor/secure-key-management.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
+# Secure use of Rekor pub key(s)
+
+A guide or document that outlines the steps for securely using Rekor public keys. Here are some possible sections for the document:
```
Comment (ltagliaferri) [inline content/en/rekor/secure-key-management.md:3]: This sentence is strange and makes me wonder how this text is being generated

**Review by ltagliaferri — CHANGES_REQUESTED** [review]
I believe the original issue is asking for a procedural guide with commands of each of these points, and there is some concern that the text is being generated. If this is not generated text, we may be able to use some of this text as a first pass, but the language should have some more detail (for example, in `## Verifying public keys` step 1 does not offer _how_  one would retrieve the public key from the TUF repository)

**Comment by haydentherapper** [comment]
There are a number of technical inaccuracies in this documentation. I dont think this addresses the issue either.

**Comment by ro4i7** [comment]
> There are a number of technical inaccuracies in this documentation. I dont think this addresses the issue either. This doesn’t have to do with storing keys, it’s about when to use a root of trust from TUF vs using a key provided out of band to verify Rekor. Like the other issue, please take a look at Rekor and follow up with an outline on the issue before writing more documentation.

Ok, I'm working on it.

**Comment by ltagliaferri** [comment]
Let's close this and continue the discussion on the issue (#72), starting with an outline

**Pull request closed without merging**